.github/workflows/release.yml

```diff
@@ -9,34 +9,8 @@ permissions:
   packages: write
 
 jobs:
-  tests:
```

Comment:
No tests need to be run before release

Reply:
There is actualy test before release, i can remove the tests job and keep the +release target with the +package sources